src/ServiceLinker/custom/New-AzServiceLinkerForContainerApp.ps1

```diff
@@ -56,27 +56,101 @@ function New-AzServiceLinkerForContainerApp {
         [System.String]
         # The name Linker resource.
         ${Name},
     
         [Parameter(Mandatory)]
         [Microsoft.Azure.PowerShell.Cmdlets.ServiceLinker.Category('Body')]
-        [Microsoft.Azure.PowerShell.Cmdlets.ServiceLinker.Models.Api20220501.IAuthInfoBase]
+        [Microsoft.Azure.PowerShell.Cmdlets.ServiceLinker.Models.Api20221101Preview.IAuthInfoBase]
         # The authentication type.
         # To construct, see NOTES section for AUTHINFO properties and create a hash table.
         ${AuthInfo},
     
         [Parameter()]
         [ArgumentCompleter([Microsoft.Azure.PowerShell.Cmdlets.ServiceLinker.Support.ClientType])]
         [Microsoft.Azure.PowerShell.Cmdlets.ServiceLinker.Category('Body')]
-        [Microsoft.Azure.PowerShell.Cmdlets.ServiceLinker.Runtime.DefaultInfo(Script='"none"')]
+        [Microsoft.Azure.PowerShell.Cmdlets.ServiceLinker.Runtime.DefaultInfo(Script = '"none"')]
         [Microsoft.Azure.PowerShell.Cmdlets.ServiceLinker.Support.ClientType]
         # The application client type
         ${ClientType},
     
-        [Parameter(Mandatory)]
+        [Parameter()]
+        [ArgumentCompleter([Microsoft.Azure.PowerShell.Cmdlets.ServiceLinker.Support.ActionType])]
+        [Microsoft.Azure.PowerShell.Cmdlets.ServiceLinker.Category('Body')]
+        [Microsoft.Azure.PowerShell.Cmdlets.ServiceLinker.Support.ActionType]
+        # Optional, indicate whether to apply configurations on source application.
+        # If enable, generate configurations and applied to the source application.
+        # Default is enable.
+        # If optOut, no configuration change will be made on source.
+        ${ConfigurationInfoAction},
+    
+        [Parameter()]
+        [Microsoft.Azure.PowerShell.Cmdlets.ServiceLinker.Category('Body')]
+        [Microsoft.Azure.PowerShell.Cmdlets.ServiceLinker.Runtime.Info(PossibleTypes = ([Microsoft.Azure.PowerShell.Cmdlets.ServiceLinker.Models.Api20221101Preview.IConfigurationInfoAdditionalConfigurations]))]
+        [System.Collections.Hashtable]
+        # A dictionary of additional configurations to be added.
+        # Service will auto generate a set of basic configurations and this property is to full fill more customized configurations
+        ${ConfigurationInfoAdditionalConfiguration},
+    
+        [Parameter()]
+        [Microsoft.Azure.PowerShell.Cmdlets.ServiceLinker.Category('Body')]
+        [Microsoft.Azure.PowerShell.Cmdlets.ServiceLinker.Runtime.Info(PossibleTypes = ([Microsoft.Azure.PowerShell.Cmdlets.ServiceLinker.Models.Api20221101Preview.IConfigurationInfoCustomizedKeys]))]
+        [System.Collections.Hashtable]
+        # Optional.
+        # A dictionary of default key name and customized key name mapping.
+        # If not specified, default key name will be used for generate configurations
+        ${ConfigurationInfoCustomizedKey},
+    
+        [Parameter(DontShow)]
```

Comment:
why you mark parameter as DontShow

Reply:
This feature is required by some specific scenario and we decide to not to expose it on powershell

Reply:
you can just comment this parameter from customized cmdlet

Reply:
Ok. The internal cmdlet has no param like this with customization directive in readme file. I remove the param as well in custom files.

Reply:
Please let me know if your PR is ready to review again by labelling `need-review`